src/functional/ffront/type_info.py

```diff
@@ -15,6 +15,13 @@
 )
 
 
+class GenericDimensions:
```

Comment:
I think this class should be defined as an empty frozen dataclass to make sure it behaves as expected